# The header printing functions incorrectly display units of MHz instead of GHz
--- /lib/sir_io.c	2009-04-13 13:53:35.000000000 -0600
+++ /lib/sir_io.c	2013-12-10 15:58:21.715268802 -0700
@@ -17,6 +17,7 @@
    modified DGL 14 Feb. 2002 + corrections to read_sir_data, added
                                sir_update_header routine
    modified DGL 16 Feb. 2002 + fixed write_sir when writing floats w/swap
+   modified RDL 5 Jan. 2012 + fixed print_head to display units in GHz
 
    Written in Ansi C 
 
@@ -1701,7 +1701,7 @@
   fprintf(omf,"  Year: %d  JD range: %d-%d",iyear,isday,ieday);
   fprintf(omf,"  Region Number: %d  Type: %d  Form: %d\n",iregion,itype,iopt);
   if (nhtype > 16) {
-     fprintf(omf,"  Polarization: %d  Frequency: %f MHz\n",ipol,ifreqhm*0.1);
+     fprintf(omf,"  Polarization: %d  Frequency: %f GHz\n",ipol,ifreqhm*0.1);
      fprintf(omf,"  Datatype: %d  Headers: %d  Ver:%d\n",idatatype,nhead,nhtype);
      fprintf(omf,"  Nodata: %f   Vmin: %f  Vmax: %f\n",anodata,v_min,v_max);
      if (ldes > 0)
@@ -1808,7 +1808,7 @@
   fprintf(omf,"  Year: %d  JD range: %d-%d",iyear,isday,ieday);
   fprintf(omf,"  Region Number: %d  Type: %d  Form: %d\n",iregion,itype,iopt);
   if (nhtype > 16) {
-     fprintf(omf,"  Polarization: %d  Frequency: %f MHz\n",ipol,ifreqhm*0.1);
+     fprintf(omf,"  Polarization: %d  Frequency: %f GHz\n",ipol,ifreqhm*0.1);
      fprintf(omf,"  Datatype: %d  Headers: %d  Ver:%d\n",idatatype,nhead,nhtype);
      fprintf(omf,"  Nodata: %f   Vmin: %f  Vmax: %f\n",anodata,v_min,v_max);
      if (ldes > 0)
